definitions.yaml

```diff
@@ -141,6 +147,12 @@ definitions:
         type: integer
         format: int32
         description: Expiration time (from creation) in hours
+      cn_prefix:
+        type: string
+        description: The common name prefix of the certificate subject.
+      o:
```

Comment:
Yeah, I was thinking about this. `o` feels super weird.

I thought to do:

`certificate_common_name_prefix` and `certificate_organizations`

or

`common_name_prefix` and `organizations`

But not sure. What do you think @puja108